docs/user_guides/projects/jobs/schedule_job.md

```diff
@@ -0,0 +1,62 @@
+---
+description: Documentation on how to schedule a job on Hopsworks.
+---
+
+# How To Schedule a Job 
+
+## Introduction
+
+Hopsworks jobs can be scheduled to run at regular intervals using the scheduling function provided by Hopsworks. Each job can be configured to have a single schedule.
+
+Schedules can be defined using the drop down menus in the UI or a [cron](https://en.wikipedia.org/wiki/Cron) expression. 
+
+!!! note "Schedule frequency"
+    The Hopsworks scheduler runs every minute. As such, the scheduling frequency should be of at least 1 minute.
+
+!!! note "Parallel executions"
+    If a job execution needs to be scheduled, the scheduler will first check that there are no active executions for that job. If there is an execution running, the scheduler will postpone the execution until the running one is done. 
+
+## UI 
+
+### Scheduling Jobs
+
+You can define a schedule for a job during the creation of the job itself or after the job has been created from the job overview UI. 
+
+<p align="center">
+  <figure>
+    <img src="../../../../assets/images/guides/jobs/job_scheduling.png" alt="Schedule a Job">
+    <figcaption>Schedule a Job</figcaption>
+  </figure>
+</p>
+
+The *add schedule* prompt requires you to select a frequency either through the drop down menus or by using a cron expression.
+You can also provide a start time to specify from when the schedule should have effect. The time will be considered as UTC time.
```

Comment:
```suggestion
You can also provide a start time to specify from when the schedule should have effect.
```

Reply:
You can mention it later, referring to all times.

Reply:
Not exactly. Later I'm talking about the start_time passed as argument to the job, not the start_time of the schedule.